StreamChatClient_v3/WebSocketClient/WebSocketPingController.swift

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+class WebSocketPingController {
+    typealias PingAction = () -> Void
+    typealias ForceReconnectAction = () -> Void
```

Comment:
I'm personally not a fan of type-aliasing simple callbacks. I find it more difficult to read because you don't from the function signature where it's used, what the signature is.

Reply:
I do that because these types are using outside.

Reply:
@b-onc , can you add your opinion? It's a tie :)

Reply:
I think it depends on the situation.
In the case of these 2, I agree that typealiasing makes using them harder, for the given reason (autocomplete doesn't show closure signature). And the typealias is not needed since the signature is not repeated.
I've also realized `ping` variable is of type `() -> Void` instead of `PingAction`, making the typealias totally useless in this case :) If the typealias is used extensively, then I'd support it, since typealiasing would save writing the signature everywhere.

Reply:
I use it like this:
```swift
        typealias CreatePingController = (
            _ timerType: Timer.Type,
            _ timerQueue: DispatchQueue,
            _ ping: @escaping WebSocketPingController.PingAction,
            _ forceReconnect: @escaping WebSocketPingController.ForceReconnectAction
        ) -> WebSocketPingController
```
```swift
    init(timerType: Timer.Type,
         timerQueue: DispatchQueue,
         ping: @escaping PingAction,
         forceReconnect: @escaping ForceReconnectAction) {
        self.timerType = timerType
        self.timerQueue = timerQueue
        self.ping = ping
        self.forceReconnect = forceReconnect
    }
```
I forgot to change it here:
```swift
    private let ping: PingAction
    private let forceReconnect: ForceReconnectAction
```

Reply:
@b-onc do you think I should remove `typealias`?

Reply:
For these 2, yes. Typealiasing closures are useful when closure signature is complex, and it's passed around many times

Reply:
This seems to more like a workaround for the API design, as mentioned here https://github.com/GetStream/stream-chat-swift/pull/373#discussion_r463581266

Reply:
I agree, it's simple closure. I removed typealias